doc/api/cli.md

```diff
@@ -617,6 +617,20 @@ files with no extension will be treated as WebAssembly if they begin with the
 WebAssembly magic number (`\0asm`); otherwise they will be treated as ES module
 JavaScript.
 
+### `--experimental-entry-url`
```

Comment:
I think we can call the flag `--entry-url`, with a printed experimental warning and the docs listing it as experimental. This isn’t a very risky feature that we need the `experimental` part in the flag name.